torch/_torch_docs.py

```diff
@@ -3643,21 +3643,23 @@ def merge_dicts(*dicts):
     tensor([ 0.5724,  0.0000, -0.1208])
 """.format(**common_args))
 
-# TODO: see https://github.com/pytorch/pytorch/issues/43667
-add_docstr(torch.linspace,
-           r"""
-linspace(start, end, steps=100, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+# TODO: update kwargs formatting (see https://github.com/pytorch/pytorch/issues/43667)
+add_docstr(torch.linspace, r"""
+linspace(start, end, steps, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
 
-Returns a one-dimensional tensor of :attr:`steps`
-equally spaced points between :attr:`start` and :attr:`end`.
+Creates a one-dimensional tensor of size :attr:`steps` whose values interpolate
+linearly from :attr:`start` to :attr:`end`, inclusive.
 
-The output tensor is 1-D of size :attr:`steps`.
+.. warning::
+    Not providing a value for :attr:`steps` is deprecated. If no value for
+    :attr:`steps` is given then a tensor with 100 elements will be created,
```

Comment:
For backwards-compatibility, not providing a value for :attr:\`steps\` will create a tensor with 100 elements. Note that this behavior is not reflected in the function signature above and should not be relied on. In a future PyTorch release...

Reply:
See below and make parallel with